packages/strapi-admin/ee/admin/containers/Roles/CreatePage/ContentTypeCollapse/CollapsePropertyMatrix/utils/tests/generateHeadersFromActions.test.js

```diff
@@ -0,0 +1,52 @@
+import generateHeadersFromActions from '../generateHeadersFromActions';
+
+describe('ADMIN | COMPONENTS | Permissions | CollapsePropertyMatrix | utils', () => {
+  it('should set isActionRelatedToCurrentProperty key to false if the applyToProperties is not an array', () => {
```

Comment:
Nitpicky: I've tend to use `when` instead of `if` in testing names. There's no `if` statement in the code (implying a condition), but there are predefined scenarios where "when" fits better from my perspectives 😊 

Not blocking at all, I'm just mentioning an opinionated thing 😝

Reply:
fixed in #9405